README_EN.md

```diff
@@ -0,0 +1,233 @@
+# Rexxar iOS
+
+[![Test Status](https://travis-ci.org/douban/rexxar-ios.svg?branch=master)](https://travis-ci.org/douban/rexxar-ios)
+[![IDE](https://img.shields.io/badge/XCode-8-blue.svg)]()
+[![iOS](https://img.shields.io/badge/iOS-7.0-green.svg)]()
+[![Language](https://img.shields.io/badge/language-ObjC-blue.svg)](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Introduction/Introduction.html)
+
+**Rexxar** is a hybrid library for mobile application development. Now it supports iOS and Android platform. `Rexxar iOS` is Rexxar's Container in iOS.
+
+With Rexxar, you can develop mobile application by traditional web technologies including javascript, html and css. Rexxar Container has not any requirements of web front side. Our demo in the web front side is implemented by [React](https://facebook.github.io/react/). You can use your own web front side framework to develop the application in Rexxar Container.
```

Comment:
Rexxar brings the ease of web technologies, including JavaScript, HTML and CSS, into mobile application development. We have used React for the web front demo, but a Rexxar Container does not confine your choice of web front end framework. You can easily replace it with [Vue](https://github.com/vuejs) or [Redux](https://github.com/reactjs/redux) [Is this true?]